src/auth/session/storage/redis.ts

```diff
@@ -108,6 +110,10 @@ export class RedisSessionStorage implements SessionStorage {
       ...this.options,
       url: this.dbUrl.toString(),
     });
+    this.client.on(
+      'error',
+      this.options.onError ? this.options.onError : () => {},
+    );
```

Comment:
I think we can merge this, but there's just one little thing I missed before - it would probably make more sense to skip setting this handler at all if `onError` isn't defined, rather than making it an empty function.

If you could make that change, we can go ahead and merge this 🙂